docs/sources/configuration/_index.md

```diff
@@ -1833,6 +1833,10 @@ the index to a backing cache store.
 # CLI flag: -<prefix>.cache.enable-fifocache
 [enable_fifocache: <boolean>]
 
+# Add tenant labels to cache-related metrics.
+# CLI flag: -<prefix>.cache.per-tenant-metrics
+[per_tenant_metrics: <boolean> | default = false]
```

Comment:
WDYT about just always doing this based on if tenancy is enabled (ie. auth = true)? Curious how much cardinality explosion we're really protecting users from here, and we already have _sooooo_ many configs.

Reply:
You know it's rude to review draft PRs :stuck_out_tongue: but thanks for taking a look

Cardinality is something that I plan to address in the description of the PR, but finishing something else up first.

Reply:
haha, it's like we're pairing...but asynchronously! yeah, I didn't take too critical an eye, but couldn't resist getting my opinions in early.